Dockerfile

```diff
@@ -16,3 +16,7 @@ RUN pipenv run python nltk_init.py
 # Install tesseract-ocr
 RUN apt-get update
 RUN apt-get install -y tesseract-ocr
+
+RUN apt-get update && \
+    apt-get -y install gcc mono-mcs && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
You dont need to run second `apt-get update` and `gcc`, `mono-mcs` packages can be just added after `tesseract-ocr`.

What is the point of deleting everything inside `/var/lib/apt/lists/`?

BTW newline at the end is missing.